uninstall.sh

```diff
@@ -1,14 +1,16 @@
 #!/bin/bash
 
 echo "This script will REMOVE the MCPI++ APT REPO! Press [Enter] to continue, or [Ctrl+C] to exit"
-read -p
+
+read -p "Waiting for choice.."
```

Comment:
Couldn't the `echo` and the `read` be merged?
```bash
read -p "This script will REMOVE the MCPI++ APT REPO! Press [Enter] to continue, or [Ctrl+C] to exit\n"
```

Reply:
Sure!